.travis.yml

```diff
@@ -33,28 +31,28 @@ before_script: |
   if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       brew update > /dev/null
       brew upgrade python  # Installs Python 3
-      brew install python@2  # Installs Python 2
       brew install scons  # Install SCons which *should* use Python 3
       pip3 install numpy cython  # Install numpy and Cython for Python 3
-      pip2 install numpy 3to2  # Install numpy and 3to2 for Python 2
       brew install boost
   else
-      pip2 install --user --install-option="--no-cython-compile" cython
-      pip2 install --user 3to2
+      pip3 install --user --upgrade pip
+      pip3 install --user --upgrade setuptools wheel
+      pip3 install --user scons
+      pip3 install --user --install-option="--no-cython-compile" cython
 
       # Install packages for the documentation
-      pip2 install --user sphinx sphinxcontrib-matlabdomain
+      pip3 install --user sphinx sphinxcontrib-matlabdomain
       # These two packages have specific bug fixes from these branches
-      pip2 install --user https://github.com/Cantera/doxylink/archive/fix_relative_path_source_14.tar.gz
-      pip2 install --user https://github.com/bryanwweber/sphinxcontrib-katex/archive/fix-options-bug.tar.gz
+      pip3 install --user https://github.com/Cantera/doxylink/archive/fix_relative_path_source_14.tar.gz
```

Comment:
With Python 3, we can use the most recent version of doxylink. Also, the bug in sphinxcontrib-katex has been fixed in the most recent release as well, so we don't need to install from GitHub.